fe/fe-core/src/main/java/com/starrocks/analysis/IndexDef.java

```diff
@@ -96,14 +96,9 @@ public void analyze() {
             throw new SemanticException("columns of index has duplicated.");
         }
 
-        if (indexType == IndexDef.IndexType.BITMAP) {
-            if (columns.size() != 1) {
-                throw new SemanticException("bitmap index can only apply to a single column.");
-            }
-        } else if (indexType == IndexDef.IndexType.GIN) {
-            if (columns.size() != 1) {
-                throw new SemanticException("INVERTED index can only apply to a single column for now.");
```

Comment:
Why remove the check?

Reply:
just use one check for gin/bitmap/ngram to avoid redundent code